elixir/apps/web/lib/web/live/policies/show.ex

```diff
@@ -30,135 +26,142 @@ defmodule Web.Policies.Show do
+          </:col>
+          <:col :let={flow} label="ACTIVITY">
+            <.link navigate={~p"/#{@account}/flows/#{flow.id}"} class={link_style()}>
+              Show
             </.link>
-          </:value>
-        </.vertical_table_row>
-      </.vertical_table>
-    </div>
+          </:col>
+        </.table>
+      </:content>
+    </.section>
 
-    <div class="grid grid-cols-1 p-4 xl:grid-cols-3 xl:gap-4 dark:bg-gray-900">
-      <div class="col-span-full mb-4 xl:mb-2">
-        <h1 class="text-xl font-semibold text-gray-900 sm:text-2xl dark:text-white">
-          Authorizations
-        </h1>
-      </div>
-    </div>
-    <.table id="flows" rows={@flows} row_id={&"flows-#{&1.id}"}>
-      <:col :let={flow} label="AUTHORIZED AT">
-        <.relative_datetime datetime={flow.inserted_at} />
-      </:col>
-      <:col :let={flow} label="EXPIRES AT">
-        <.relative_datetime datetime={flow.expires_at} />
-      </:col>
-      <:col :let={flow} label="CLIENT (IP)">
-        <.link
-          navigate={~p"/#{@account}/clients/#{flow.client_id}"}
-          class="font-medium text-blue-600 dark:text-blue-500 hover:underline"
-        >
-          <%= flow.client.name %>
-        </.link>
-        (<%= flow.client_remote_ip %>)
-      </:col>
-      <:col :let={flow} label="GATEWAY (IP)">
-        <.link
-          navigate={~p"/#{@account}/gateways/#{flow.gateway_id}"}
-          class="font-medium text-blue-600 dark:text-blue-500 hover:underline"
+    <.section>
+      <:title>Danger Zone</:title>
+      <:action>
+        <.action_button
+          type="danger"
+          icon="hero-trash"
+          phx-click="delete"
+          phx-value-id={@policy.id}
+          data-confirm="Are you sure you want to delete this policy?"
         >
-          <%= flow.gateway.group.name_prefix %>-<%= flow.gateway.name_suffix %>
-        </.link>
-        (<%= flow.gateway_remote_ip %>)
-      </:col>
-      <:col :let={flow} label="ACTIVITY">
-        <.link
-          navigate={~p"/#{@account}/flows/#{flow.id}"}
-          class="font-medium text-blue-600 dark:text-blue-500 hover:underline"
-        >
-          Show
-        </.link>
-      </:col>
-    </.table>
-
-    <.header>
-      <:title>
-        Danger zone
-      </:title>
-      <:actions>
-        <.delete_button phx-click="delete" phx-value-id={@policy.id}>
           Delete Policy
-        </.delete_button>
-      </:actions>
-    </.header>
+        </.action_button>
+      </:action>
+      <:content></:content>
+    </.section>
     """
   end
 
+  # Event handler functions
+
   def handle_event("delete", %{"id" => _policy_id}, socket) do
     {:ok, _} = Policies.delete_policy(socket.assigns.policy, socket.assigns.subject)
     {:noreply, push_navigate(socket, to: ~p"/#{socket.assigns.account}/policies")}
   end
+
+  # Private/Helper Functions
+
+  defp pretty_print_date(date) do
+    "#{date.month}/#{date.day}/#{date.year} #{date.hour}:#{date.minute}:#{date.second}"
```

Comment:
We have a `<.datetime />`  and `<.relative_datetime />` for that.

Reply:
Good to know.  This function was just being moved from up above and at the time it was written I'm not sure if those components existed (or perhaps I just didn't realize they existed at the time 😄 ) 

I'll make sure to update this page to use those components and then we can get rid of this function all together.

Reply:
This function has been removed and the `<.datetime>` component has been used in it's place